src/js/languages/default.json

```diff
@@ -58,6 +58,7 @@
   "dataTable": {
     "ascending": "ascending",
     "descending": "descending",
+    "resizerAria": "Resize {headerText} column",
```

Comment:
do we need to add to proptypes and TS?

Reply:
Good catch, just pushed up a commit to address this

Reply:
thanks! I know the behavior might be hard to test should we just add a test for adding a `message` for `resizerAria` ?

Reply:
oh I meant like passing a custom `message` through grommet `messages` prop just making sure it comes through. I caught a mistake in my PR I have so figured more of a nice to have not necessary though

Reply:
Ah gotcha, I can add a test for that

Reply:
Just pushed up a test